blaze-client/src/main/scala/org/http4s/blaze/client/BlazeClientBuilder.scala

```diff
@@ -368,6 +428,7 @@ object BlazeClientBuilder {
       asynchronousChannelGroup = None,
       channelOptions = ChannelOptions(Vector.empty),
       customDnsResolver = None,
+      retries = 2,
```

Comment:
Not a blocker at all, I'm just interested in why '2' was chosen here?

Reply:
1. It matches Ember.
2. I think Ember matches an RFC, but damn if I can find it in there now.

Reply:
Got it, thanks!